Cargo.toml

```diff
@@ -12,8 +12,8 @@ readme = "README.md"
 lazy_static = "0.2"
 
 [target.'cfg(any(target_os = "macos", target_os = "ios"))'.dependencies]
-security-framework = { version = "0.1.15", features = ["OSX_10_8" ]}
-security-framework-sys = "0.1.15"
+security-framework = { git = "https://github.com/scottschroeder/rust-security-framework", branch = "noverify", features = ["OSX_10_8" ]}
+security-framework-sys = { git = "https://github.com/scottschroeder/rust-security-framework", branch = "noverify" }
```

Comment:
This PR is blocked on updates to the `security-framework` crate, I'll update the `Cargo.toml` if/when a new crate gets published there.